odoo/tools/translate.py

```diff
@@ -1098,43 +1067,14 @@ def process_row(row):
             dic = dict.fromkeys(('type', 'name', 'res_id', 'src', 'value',
                                  'comments', 'imd_model', 'imd_name', 'module'))
             dic['lang'] = lang
-            dic.update(zip(fields, row))
+            dic.update(row)
 
             # do not import empty values
             if not env.context.get('create_empty_translation', False) and not dic['value']:
                 return
 
-            if use_pot_reference:
-                # discard the target from the POT targets.
-                src = dic['src']
-                target_key = (dic['type'], dic['name'], dic['type'] != 'code' and dic['res_id'] or 0)
-                target = pot_targets.get(src)
-                if not target or target_key not in target.targets:
-                    _logger.info("Translation '%s' (%s, %s, %s) not found in reference pot, skipping",
```

Comment:
@d-fence good news, this PR removes this message that spam your runbot logs :wink:

Reply:
I hope that it will be merged one day :crossed_fingers: